oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/NodeStateEntryBatch.java

```diff
@@ -51,49 +49,46 @@ public static NodeStateEntryBatch createNodeStateEntryBatch(int bufferSizeBytes,
     }
 
     private final ByteBuffer buffer;
-    private final ArrayList<SortKey> sortBuffer;
     private final int maxEntries;
+    private int numberOfEntries = 0;
```

Comment:
0 is the default
```suggestion
    private int numberOfEntries;
```

Reply:
As discussed privately, I prefer to initialize the values explicitly, to convey the intention that 0 is the initial value for the field, as opposed to a variable that is not assigned in its definition because it will be initialized in the construtor.